roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2

```diff
@@ -81,6 +81,9 @@ spec:
 {% if kube_api_anonymous_auth is defined and kube_version | version_compare('v1.5', '>=')  %}
     - --anonymous-auth={{ kube_api_anonymous_auth }}
 {% endif %}
+{% for flag in apiserver_custom_flags %}
```

Comment:
There are so few lists in params that  I am worried about users trying to use this with a plain string and then it will be split in to a list of chars. This is an alternative that will handle both string or list:
```
{% if apiserver_custom_flags is string %}
    - {{ apiserver_custom_flags }}
{% else %}
  {% for flag in apiserver_custom_flags %}
    - {{ flag }}
  {% endfor %}
{% endif %}
```

What are your thoughts?